packages/buildcop/src/buildcop.ts

```diff
@@ -278,40 +291,44 @@ buildcop.formatFailure = (failure: TestFailure): string => {
   return `${pkg}: ${failure.testCase} failed`;
 };
 
-buildcop.findFailures = (xml: string): TestFailure[] => {
-  const obj = xmljs.xml2js(xml);
-  const failures: TestFailure[] = [];
-  for (const suites of obj.elements) {
-    if (suites.name !== 'testsuites') {
-      continue;
+buildcop.findTestResults = (xml: string): TestResults => {
+  const obj = xmljs.xml2js(xml, { compact: true }) as xmljs.ElementCompact;
+  const failures: TestCase[] = [];
+  const passes: TestCase[] = [];
+  // Python doesn't always have a top-level testsuites element.
```

Comment:
I'll be curious to see if we bump into similar edge-cases with Node.js.

Reply:
Me too. I imagine we will for all of the languages, which is why I included comments/tests for both languages.